.github/workflows/documentation.yml

```diff
@@ -0,0 +1,21 @@
+name: documentation
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'doc/*'
+
+jobs:
+  build:
+    continue-on-error: true
+    runs-on: ubuntu-latest
+    container: "lammps/buildenv:fedora32_mingw"
+    steps:
+      - uses: actions/checkout@master
+      - name: Generate HTML
```

Comment:
Unless these things depend on each other, I would make this a matrix then it is easier to see if html, pdf or spelling fails.